### PR TITLE
doc groonga-httpd: add a explanation about access log

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: doc/source/reference/executables/groonga-httpd.rst [hunk 1]
```diff
@@ -543,6 +543,41 @@ configuration, run the following command to remove cache files::
...
+  http {
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  access.log  main;
+  }
+
+We can specify log format by ``log_format`` directive.
+For the detailes of available items of the log,
```
Comment (kenhys) [inline doc/source/reference/executables/groonga-httpd.rst]: detailes ->
details

**Comment by komainu8** [comment]
@kenhys Thank you for your comment.
I've fixed it.

File: doc/source/reference/executables/groonga-httpd.rst [hunk 2]
```diff
@@ -543,6 +543,41 @@ configuration, run the following command to remove cache files::
 If you use Groonga's query cache feature, you don't need to expire
 cache by hand. It is done automatically.
 
+Nginx log related directives
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``access_log``
+""""""""""""""
+
+We can output Nginx's log by the ``access_log`` directive.
```
Comment (kou) [inline doc/source/reference/executables/groonga-httpd.rst]: `nginx`

File: doc/source/reference/executables/groonga-httpd.rst [hunk 3]
```diff
@@ -543,6 +543,41 @@ configuration, run the following command to remove cache files::
...
+The following example is default Nginx's log settings::
+
+  http {
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
```
Comment (kou) [inline doc/source/reference/executables/groonga-httpd.rst]: Why do you use `main` as format name?
The default format is called `combined` in nginx.

File: doc/source/reference/executables/groonga-httpd.rst [hunk 4]
```diff
@@ -543,6 +543,41 @@ configuration, run the following command to remove cache files::
...
 cache by hand. It is done automatically.
 
+Nginx log related directives
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``access_log``
+""""""""""""""
+
+We can output Nginx's log by the ``access_log`` directive.
+The following example is default Nginx's log settings::
```
Comment (kou) [inline doc/source/reference/executables/groonga-httpd.rst]: the default

File: doc/source/reference/executables/groonga-httpd.rst [hunk 5]
```diff
@@ -543,6 +543,41 @@ configuration, run the following command to remove cache files::
...
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  access.log  main;
+  }
+
+We can specify log format by ``log_format`` directive.
+For the details of available items of the log,
+see http://nginx.org/en/docs/http/ngx_http_log_module.html.
```
Comment (kou) [inline doc/source/reference/executables/groonga-httpd.rst]: Use link markup for the URL.

File: doc/source/reference/executables/groonga-httpd.rst [hunk 6]
```diff
@@ -543,6 +543,41 @@ configuration, run the following command to remove cache files::
...
+
+Nginx's access log is outputted when processing ends.
```
Comment (kou) [inline doc/source/reference/executables/groonga-httpd.rst]: when a requests is processed.

File: doc/source/reference/executables/groonga-httpd.rst [hunk 7]
```diff
@@ -543,6 +543,41 @@ configuration, run the following command to remove cache files::
...
+If we want to output the request processing time,
+we need to set configuration as below::
+
+  http {
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for" '
+                      '"$request_time" ';
```
Comment (kou) [inline doc/source/reference/executables/groonga-httpd.rst]: Remove a needless white space after `"$request_time"`.

**Comment by komainu8** [comment]
@kou Thank you for your comments.
I've fixed them.

**Pull request merged**